ts/src/checker.ts

```diff
@@ -350,4 +351,49 @@ export class LicenseChecker extends EventEmitter {
     }
     this.emit('end');
   }
+
+  /** set default event Handlers for CLI output. */
+  defaultHandlers() {
```

Comment:
I'd prefer the name, `setDefaultHandlers()`.

Reply:
Done.